src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -1,35 +1,32 @@
 (ns status-im.ui.screens.chat.message.message
-  (:require [quo.core :as quo]
```

Comment:
this file hasn't been changed , it was reamed, and moved, that why git shows changes , but its just different file